components/ucd/normal/tests/unicode_version_tests.rs

```diff
@@ -20,3 +21,149 @@ fn test_version_against_ucd_core() {
         unic_ucd_core::UNICODE_VERSION
     );
 }
+
```

Comment:
Whoops, my bad. If you want to preserve the state of `tests/unicode_version_tests.rs`, I can (look up how to and) squish the history of the PR to one commit. For now I'm just adding in another commit to move the code.

I'm doing this first, so it may break the review integration a little, but ¯\\\_(ツ)_/¯ I'm addressing the points anyway.